content/source/docs/enterprise/before-installing/postgres-requirements.html.md

```diff
@@ -7,37 +7,41 @@ page_title: "PostgreSQL Requirements - Before Installing - Terraform Enterprise"
 
 -> **Note:** These requirements apply to the external services operational mode, not the managed disk operational mode. See the [Pre-Install Checklist](./index.html) for more information.
 
-When Terraform Enterprise uses an external PostgreSQL database, the
-following must be present on it:
+To use an external PostgreSQL database with Terraform Enterprise, the following
+requirements must be met:
 
-* PostgreSQL version `>= 9.4` and `< 10.0`
-* User with the ability to create/modify/read tables and indices on all schemas created
-  * If it's not feasible to have a user with "CREATE EXTENSION", then create the [extensions](#extensions) below before installation
-* The following PostgreSQL schemas must be installed into the database: `rails`, `vault`, `registry`
+* The PostgreSQL Server version must be `>= 9.4` and `< 11.0`.
```

Comment:
Not sure if we're ready to bump the version to include 10?

Reply:
I did that based on previous input from @evanphx.

Reply:
Cool, nice to hear 🙌